[16.0][ADD] crm_create_project: Allow create project from lead or opportunity (#577)
OCA/crm: crm_project_create/readme/USAGE.md

To use this module, you need to:

1. Go to CRM > Sales > My Pipeline
1. Create a new opportunity or join an existing one.
1. Click on **Create project** in header.
1. A wizard is shown to fill project name and/or project description.
1. Click on **Create project**.
1. You can see project created in page **Extra Information**, inside group **Misc**.

> 1. Is you has not group *Project: Administrator* you can edit project after creating.

Reviewer: ```suggestion
1. Is you have *Project: Administrator* permissions, you can edit the project after creating it.
```
